src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+
+  sourceString
+    .split(';')
+    .filter(el => el.trim() !== '')
+    .forEach(css => {
+      const [code, value] = css.split(':');
+
+      cssProperties[code.trim()] = value.trim();
+    });
+
+  return cssProperties;
```

Comment:
That is great! But I believe that you can find a solution using `Array.prototype.reduce` instead of `forEach`